Traceback (most recent call last):
  File "/Users/star/Projects/Python Projects/Automate the Boring Stuff with Python/Chapter 10/Debugging/traceback_format.py", line 4, in <module>
    raise Exception('This is the error message.')
Exception: This is the error message.
